protoconfloader.go

```diff
@@ -114,10 +114,6 @@ func (c *Configuration) LoadConfig(configPath string, configName string) error {
 // It watches for file changes and agent updates using separate goroutines.
 // The method logs the successful start of watching and returns nil upon successful completion.
 func (c *Configuration) WatchConfig(ctx context.Context) error {
-	if !c.isLoaded.Load() {
```

Comment:
Why did you remove this lines?

Reply:
@smintz ?

Reply:
Some users might only use the agent for subscription. In these cases, this statement prevents watching the config for no good reason.